src/webapp/project_reader.py

```diff
@@ -65,6 +65,14 @@ def get_one_project(file: str, campus_grid_ids: Dict, vos_data: VOsData) -> Dict
             data['ResourceAllocations'] = {"ResourceAllocation": resource_allocations}
         if 'ID' not in data:
             del project['ID']
+
+        name_from_filename = os.path.basename(file)[:-5]  # strip '.yaml'
```

Comment:
What about `.yml`?

Reply:
We don't read .yml files; this goes for the VO and resource readers as well.